### DISCUSSION TITLE
Inconsistent Observation Data across different requests

### DISCUSSION THREAD

**Discussion by penguinrunner** [weather-gov/api#334] [Q&A]
I am getting inconsistent data for station observations depending on how I request it from the API

https://api.weather.gov/stations/KMFD/observations/latest
https://api.weather.gov/stations/KMFD/observations?limit=1
https://api.weather.gov/stations/KMFD/observations?limit=4
https://api.weather.gov/stations/KMFD/observations

It is currently 11:25 EST / 15:25 UTC

When I request the first API call, I receive an observation for: 2020-06-18T12:52:00+00:00, and the temp is Null

When I request the second API call, I receive an observation for : 2020-06-18T11:52:00+00:00, and the temp is also Null

The third API call returns observations for the following:
2020-06-18T14:52:00+00:00, Temp is null
2020-06-18T13:52:00+00:00 Temp is valid
2020-06-18T12:52:00+00:00 Temp is valid
2020-06-18T11:52:00+00:00 Temp is valid

The fourth call starts with the observation for 13:52, but doesn't have the 14:52 data.

I assumed /latest would pull the most recent observation, but it is not doing so.

Also temperatures are showing Null when pulled via latest or limit=1, but are valid when limit is set greater than 1.

The station is reporting to the website fine at https://forecast.weather.gov/MapClick.php?lat=40.8803&lon=-82.6601. But at the current time it has data for the temperature on the 14:52 observation that I can't pull with latest, and when I do get it using limit=4, the temp is Null, but on the site its showing.

**Comment by hawtkey** [comment]
After fixing the type error in #65, which had caused my query to flat-line about 26hrs prior, the data flat-lined again at around 10:00 PST 6/18. New friends working at NOAA maybe?

Current local time is ~19:00 PST 6/18. 

If I navigate to KPDX/observations/latest the timestamp is now stuck in the past at 15:53, with the same dead data from 10:00. If I navigate to the 0 cell in KPDX/observations the timestamp is in the future, 23:53 PST. The actual latest observation is now 00:53 6/19 by going direct to KPDX/observations/2020-06-19T00:53:00+00:00

Looking at KSFO in the same timezone, latest is stuck at 21:56 and the 0 cell is showing 00:56 6/19

**Comment by StephenClouse** [comment]
We upgraded the API over the last week and we are still investigating what may be a bug that is causing at least some of this. Please bear with us for a bit.
